### PR TITLE
Update TSC members and policies/procedures

### PR DESCRIPTION
**Pull request by brianwarner**
This commit adds the current TSC members, and cleans up some aspirational
statements because the actions have already been completed.

Signed-off-by: Brian Warner <brian@bdwarner.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,18 @@ The Presto Foundation maintains a [public calendar](https://calendar.google.com/
 
 ## Members
 
-The TSC voting members are initially the [Project’s Committers](https://github.com/prestodb/presto/wiki/committers), and the current TSC chair is Nezih Yigitbasi ([@nezihyigitbasi](https://github.com/nezihyigitbasi)). At the inception of the project, the Committers of the Project will be as set forth within the "CONTRIBUTING" file within the Project’s code repository
+The TSC voting members are initially the [Project’s Committers](https://github.com/prestodb/presto/wiki/committers).
```
Comment (caniszczyk) [inline README.md]: can we no longer use the wiki page for maintainers

1) can we add affiliations
2) can we add term lengths/dates

Comment (brianwarner) [reply README.md]: Yes, good point. I think it makes sense to maintain the list of committers in CONTRIBUTING.md in .github, so it replicates across all repos, and so were only tracking that info in one place. I'll fix this link.

I can also add term lengths and affiliations for the TSC members in this file (even though they're initially the same).

Comment (brianwarner) [reply README.md]: I've just pushed this in 62028e2.  A subsequent commit (5618251) contains a proposal for staggering term lengths to encourage continuity during and after elections.

**Comment by brianwarner** [comment]
In addition to the prior updates, I've also pushed some proposed policies and procedures for the TSC to review and consider for adoption.  

The Technical Charter provides the TSC a great deal of flexibility to set its own governance.  To help this process along, I've created a draft which the TSC members can review and modify as necessary.

I will add this to the upcoming meeting agenda.

File: README.md [hunk 2]
```diff
@@ -22,9 +22,68 @@ Once you join the The [Presto Foundation Slack Workspace](https://join.slack.com
...
+
+The Presto TSC is governed by the [Presto Technical Charter](CHARTER.md).  The Charter provides foundational structure for the TSC on topics such as its scope, how to make decisions, and how to make changes to itself.  At the same time, it grants the TSC a high degree of freedom when determining how to implement the policies of the Presto Foundation.
+
+The following policies and procedures have been adopted by the TSC.
+
+### Term of TSC voting members
+
+The Presto TSC seeks to balance the energy and enthusiasm of new members with the stability provided by long-term members.  As such, TSC voting members will serve two-year terms, after which they will be eligible for re-election.  There is no limit to the number of times an individual may run for an open seat.
+
+To provide balance, the terms of TSC members will be adjusted so that roughly 50% of seats are elected each year.  If the classes become unbalanced, then the TSC shall designate the appropriate number of seats as one-year terms at the first opportunity to fix the imbalance.
```
Comment (zhenxiao) [inline README.md]: re-elect 1/3 of the TSC seats, so that TSC is stable?

File: README.md [hunk 3]
```diff
@@ -22,9 +22,68 @@ Once you join the The [Presto Foundation Slack Workspace](https://join.slack.com
...
+
+For simplicity, membership terms will be aligned to the calendar year, and will run from January 1st to December 31st of the following year. Initially, the TSC shall randomly designate 50% of the seats to be one-year terms, and the remainder shall be two-year terms.
+
+### Becoming a TSC Member
+
+The Technical Charter specifies that TSC voting members are initially the Presto committers.  Prior to each election, the TSC will determine the number of available seats and solicit nominations from the community.  The TSC has primary responsibility for ensuring there are sufficient candidates to fill each available seat.
+
+#### Eligibility
+
+Beginning with the 2021 election, any member of the technical community may stand for election to the TSC.  Membership in the Presto Foundation or status as a Presto committer are not pre-requisites.
```
Comment (zhenxiao) [inline README.md]: still think committer is a pre-requisite for TSC member

File: README.md [hunk 4]
```diff
@@ -22,9 +22,68 @@ Once you join the The [Presto Foundation Slack Workspace](https://join.slack.com
...
+| Masha Basmanova           |[mbasmanova](https://github.com/mbasmanova)          | Facebook    | 1 Jan, 2020  | TBD          |
+| Zhenxiao Luo              |[zhenxiao](https://github.com/zhenxiao)              | Twitter     | 1 Jan, 2020  | TBD          |
+| Andrii Rosa               |[arhimondr](https://github.com/arhimondr)            | Facebook    | 1 Jan, 2020  | TBD          |
+| Rebecca Schlussel         |[rschlussel](https://.github.com/rschlussel)         | Facebook    | 1 Jan, 2020  | TBD          |
+| James Sun                 |[highker](https://github.com/highker)                | Facebook    | 1 Jan, 2020  | TBD          |
+| Wenlei Xie                |[wenleix](https://github.com/wenleix)                | Facebook    | 1 Jan, 2020  | TBD          |
+| Rongrong Zhong            |[rongrong](https://github.com/rongrong)              | Facebook    | 1 Jan, 2020  | TBD          |
+| Shixuan Fan               |[shixuan-fan](https://github.com/shixuan-fan)        | Facebook    | 1 Jan, 2020  | TBD          |
+| Jiexi Lin                 |[jessesleeping](https://github.com/jessesleeping)    | Facebook    | 1 Jan, 2020  | TBD          |
```
Comment (aweisberg) [inline README.md]: Leiqing is also a committer now

File: README.md (hunk 2 above)
Comment (brianwarner) [reply README.md]: Either way is fine, although if all terms are the same length it would imply 3 year terms.  I might recommend starting with "how long should the a TSC serve before standing for reelection?" and working backwards from there.

File: README.md (hunk 3 above)
Comment (brianwarner) [reply README.md]: May I suggest we add this to the agenda for the next TSC call (unless folks would like to discuss here as comments)?  I believe there are a range of opinions on this particular topic, and it would be good to reach an agreement.

File: README.md (hunk 2 above)
Comment (zhenxiao) [reply README.md]: sure. nice

File: README.md (hunk 3 above)
Comment (zhenxiao) [reply README.md]: sure. go ahead

File: README.md [hunk 5]
```diff
@@ -42,6 +42,7 @@ The current members of the Presto TSC are:
 | Rongrong Zhong            |[rongrong](https://github.com/rongrong)              | Facebook    | 1 Jan, 2020  | TBD          |
 | Shixuan Fan               |[shixuan-fan](https://github.com/shixuan-fan)        | Facebook    | 1 Jan, 2020  | TBD          |
 | Jiexi Lin                 |[jessesleeping](https://github.com/jessesleeping)    | Facebook    | 1 Jan, 2020  | TBD          |
+| Venki Korukanti           |[vkorukanti](https://github.com/vkorukanti)          | Uber        | 14 Feb, 2020  | TBD         |
```
Comment (nezihyigitbasi) [inline README.md]: Given that TSC approved adding Venki, this can be a separate PR and can be merged. The rest of the PR is more complicated and may need more time to review.

**Comment by galemccommons** [comment]
Merging. Thanks @brianwarner

**Comment by galemccommons** [comment]
Apologies, that was a premature merge.  We still have some discussion points, I’ll revert it for the time being
